srcpkgs/Tuba/template

```diff
@@ -1,14 +1,21 @@
 # Template file for 'Tuba'
 pkgname=Tuba
-version=0.2.0
+version=0.3.2
 revision=1
 build_style=meson
 configure_args="-Ddistro=true"
-hostmakedepends="pkg-config gettext desktop-file-utils vala glib-devel"
-makedepends="libadwaita-devel libgee-devel libsecret-devel gtksourceview5-devel"
+hostmakedepends="desktop-file-utils gettext pkg-config"
+makedepends="gtksourceview5-devel libadwaita-devel libgee-devel libsecret-devel
+ libwebp-devel"
+depends="webp-pixbuf-loader"
+checkdepends="appstream-glib"
 short_desc="Browse the fediverse"
 maintainer="Michal Vasilek <michal@vasilek.cz>"
 license="GPL-3.0-only"
-homepage="https://tuba.geopjr.dev/"
-distfiles="https://github.com/GeopJr/Tuba/archive/refs/tags/v${version}.tar.gz"
-checksum=c698206bbbd06a1ce5a6e8cc9e6590411dc1cce1dde87fb0e34d8e35432ee2b1
+homepage="https://github.com/GeopJr/Tuba"
+distfiles="${homepage}/archive/v${version}.tar.gz"
```

Comment:
no, revert to the previous version

Reply:
Reverted. Could you explain why? Is the old version of a `homepage` variable more preferable? Is the old version of a `distfiles` variable more preferable?

Reply:
This is useful! I will keep the diff small and clean in the future!

P.S. Doesn't this piece of information need to be in [CONTRIBUTING.md](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md)?

Reply:
I am not sure what exactly should be added there, keeping commits clean and not doing unrelated or unnecessary changes is just a basic git best practice.